lib/feature_management.rb

```diff
@@ -129,6 +129,6 @@ def self.logo_upload_enabled?
   end
 
   def self.log_to_stdout?
-    !Rails.env.test? && Figaro.env.log_to_stdout?
```

Comment:
oh yikes --- I should have looked into the `?` methods more closely when reviewing the last PR, I thought they would behave more similarly

Reply:
To clarify, is the issue here that since all values in the `Figaro.env` hash are strings, trying to coerce a string value `'false'` to a boolean would always return `true`?

```
$ irb
irb(main):005:0> !! 'false'
=> true
```

Reply:
Yeah, or even if the value was `''` , then `!!'' == true`